bifacial_radiance/main.py

```diff
@@ -3247,10 +3249,41 @@ def generate_spectral_tmys(self, wavelengths, weather_file, location_name, spect
             output_folder = os.path.join('data','spectral_tmys')
         if not os.path.exists(output_folder):
             os.makedirs(output_folder, exist_ok=True)
+
+        
         
         su.generate_spectral_tmys(wavelengths=wavelengths, spectra_folder=spectra_folder,
-                                  weather_file=weather_file, location_name=location_name,
+                                  metdata=self.metdata, location_name=location_name,
                                   output_folder=output_folder)
+        
+    def integrated_spectrum(self, spectra_folder):
```

Comment:
@mjprilliman  - we should come up with another pytest in test_spectra.py that exercise this integrated_spectrum function.   I have one on my local computer that can be added to `development` once this gets merged...